### PR TITLE
Fixes #32079 - CI reporting for pulpcore integration tests

### PR DESCRIPTION
**Pull request by jjeffers**

### CODE REVIEW

File: lib/katello/tasks/jenkins.rake [hunk 1]
```diff
@@ -18,6 +18,13 @@ begin
       Rake::Task['jenkins:setup:minitest'].invoke
       Rake::Task['rake:test:katello:test'].invoke
     end
+
+    desc "Runs puplcore integration tests"
+    task 'katello:pulpcore' do
+      Rake::Task['jenkins:setup:minitest'].invoke
+      Rake::Task['rake:test:katello:pulpcore'].invoke
+    end
```
Comment (jlsherrill) [inline lib/katello/tasks/jenkins.rake:26]: should these just always run in live mode?

Comment (jjeffers) [reply lib/katello/tasks/jenkins.rake:26]: Not necessarily - the setup is primarily for generation of the JUnit style output, so this mirrors the same pulpcore test task in test.rake. The advantage here is that we are only interested in the pulp3 tests. In the pipeline, we issue the command:
`mode=all bundle exec rails jenkins:katello:pulpcore`.

**Review by jlsherrill — APPROVED** [review]

**Comment by jjeffers** [comment]
[test katello]

**Comment by jjeffers** [comment]
[test katello]

**Comment by jjeffers** [comment]
[test katello]

**Comment by jjeffers** [comment]
[test katello]

**Comment by jjeffers** [comment]
[test katello]

**Pull request merged**